code/datums/supply_packs/clothing.dm

```diff
@@ -180,91 +77,19 @@
 		/obj/item/clothing/accessory/storage/holster,
 		/obj/item/clothing/accessory/storage/holster
 	)
-	cost = RO_PRICE_VERY_CHEAP
+	cost = 20
 	containertype = /obj/structure/closet/crate/green
 	containername = "Shoulder Holster Crate"
 	group = "Clothing"
 
-
-/datum/supply_packs/gun_holster
-	contains = list(
-					/obj/item/storage/large_holster/m39,
-					/obj/item/storage/large_holster/m39
-					)
-	name = "M39 Belt Holster Crate (x2)"
-	cost = RO_PRICE_VERY_CHEAP
-	containertype = /obj/structure/closet/crate
-	containername = "M39 Belt Holster Crate"
-	group = "Clothing"
-
 /datum/supply_packs/gunslinger_holster
 	contains = list(
 					/obj/item/storage/belt/gun/m44/gunslinger,
 					/obj/item/storage/belt/gun/m44/gunslinger
 					)
 	name = "Red Ranger Cowboy Gunbelt Crate (x2)"
-	cost = RO_PRICE_VERY_CHEAP
+	cost = 20
 	contraband = 1
 	containertype = /obj/structure/closet/crate
 	containername = "Cowboy Costume Crate"
 	group = "Clothing"
-
-/datum/supply_packs/gun_holster/m44
-	name = "M44 Belt Holster Crate (x2)"
-	containername = "M44 Belt Holster Crate"
-	cost = RO_PRICE_WORTHLESS
-	contains = list(
-					/obj/item/storage/belt/gun/m44,
-					/obj/item/storage/belt/gun/m44
-					)
-	group = "Clothing"
-
-/datum/supply_packs/gun_holster/m4a3
-	name = "M4A3 Belt Holster Crate (x2)"
-	containername = "M4A3 Belt Holster Crate"
-	cost = RO_PRICE_WORTHLESS
-	contains = list(
-					/obj/item/storage/belt/gun/m4a3,
-					/obj/item/storage/belt/gun/m4a3
-					)
-	group = "Clothing"
-
-//*******************************************************************************
-//CLOTHING
-//*******************************************************************************/
-
-
-/datum/supply_packs/officer_outfits//lmao this shit is so hideously out of date
```

Comment:
i think we should keep this one too, it has shenanigans potential

Reply:
ok